### ISSUE TITLE
Data splits

### ISSUE THREAD

**Issue by shara19** [vagarwal87/saluki_paper#13]
Hi,

How was the data divided into different splits (ie, train, valid, test)? I don't think this is clear in the paper, unless I missed it. Apologies if I overlooked it. I saw the `genes.tsv` in each fold directory that has the mapping of gene to split but I'm more interested in understanding if theres any specific regions/logic/script used to split the datasets into splits? 

I did a basic check to see if they are split based on chromosome but looks like that's not the case as the test set genes seem to come from all the chromosomes.

The reason Im asking this is because I'm considering finetuning Saluki on a different task and wanted to make sure that I'm not try to leak any data by using the wrong splits for my training dataset.

Thanks!

**Comment by shara19** [comment]
So if Im understanding correctly, the paper says homologous genes were used to split the data. Based on basenji's pre-processing scripts, this meant dividing the training data into contiguous regions and sampling from them, right? If so, are these contigs stored anywhere?

**Comment by shara19** [comment]
I guess I could essentially calculate the above using genes.tsv. 

Can you confirm this is the right approach?

PS, sorry about the deluge of notes. 🙃

**Comment by davek44** [comment]
I believe we simply divided the human genes randomly into 10 folds. When we added joint training on mouse, we ensured that mouse genes were divided into the same fold as their human homologues. There was no region or contig or chromosome consideration.

**Comment by shara19** [comment]
Alright, thanks for the clarification!
